packages/icon-magic/README.md

```diff
@@ -0,0 +1,3 @@
+This package is just a pass through to the [Icon Magic CLI](https://www.npmjs.com/package/@icon-magic/cli)
+
+Follow the steps there to install and use.
```

Comment:
We will need to specify installation instructions here as they're different than that of the cli.